src/main/java/org/sonar/plugins/csharp/CSharpCodeCoverageProvider.java

```diff
@@ -135,7 +135,7 @@ public CSharpCoverageAggregator(Settings settings) {
   public static class CSharpCoverageReportImportSensor extends CoverageReportImportSensor {
 
     public CSharpCoverageReportImportSensor(CSharpCoverageAggregator coverageAggregator, FileSystem fs) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "fs". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1172)